.pre-commit-config.yaml

```
@@ -0,0 +1,9 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.1.4
```

Update to latest version
```suggestion
  rev: v0.1.7
```